scripts/build_all_oval_definitions.py

```diff
@@ -52,7 +56,7 @@ def main():
             OvalGenerator.queue_element_file(element_type, file_path)
 
         # write output file
-        outfile = './tmp/gen.{0}'.format(lib_repo.oval_id_to_path(def_id))
+        outfile = '{1}/gen.{0}'.format(lib_repo.oval_id_to_path(def_id), tempfile.gettempdir())
```

Comment:
Hi Noam, 

I don't think this script is used and can't remember why I created it. 

People generally use build_all_oval_definitions.py which allows you to specify where to put the generated oval.xml file and has a variety of filtering and validation options. It also has an option to create "all_definitions", which would seem to obviate the need for this script.

I suggest we remove this script entirely.

Reply:
Most people use build_oval_definitions_file, and not this one, I used it in order to debug some code I added - but I don't use it as well.